docs/plantilla-para-aplicacion-web.md

```diff
@@ -7,6 +7,43 @@ sidebar_label: Plantilla para Aplicación Web
 ## Plantilla para Aplicación Web de React
 
 Plantilla para aplicaciones eb de React incorporando UAL para autenticadores EOSIO (wallets)
```

Comment:
Error en la palabra **eb**, en la oración "Plantilla para aplicaciones eb de React incorporando UAL para autenticadores EOSIO (wallets)"

Reply:
Thanks Leister, this error fix in other PR please or contributes in this PR and fix error